### ISSUE TITLE
less stiff hair and reshading

### ISSUE THREAD

**Issue by PeachyPieeee** [Monika-After-Story/MonikaModDev#9215]
hello! i’d like to contribute to the mod. i’ve reshaded the front part of monika’s hair and to make it less stiff. i’m not sure how to submit a pull request for art as i have no experience in coding, but i’ll submit my contribution here. if it were to be implemented, or needs changing and more editing, please do not hesitate to use it or contact me :)
![8036BD20-DCE0-41E4-A2FB-69AF4C59B199](https://user-images.githubusercontent.com/66063309/172284672-573d5e61-7276-4996-afcc-0cfddc5c9404.png)

**Comment by kkrosie123** [comment]
Hello, PeachyPie! This is really nice looking. Is it OK to see the full version of what you would do for the bangs?

Secondly, I should just let you know, these bangs would need to be updated for **every hairstyle and spritepack.** If you’re alright with that, that's absolutely wonderful! Me and the rest of the team absolutely adore your edits thus far.

Excited to see what you do! Thank you for your submission!

**Comment by PeachyPieeee** [comment]
![image](https://user-images.githubusercontent.com/66063309/172812860-33834b90-aa4b-42f9-b5b1-b2e15c8bc866.jpeg)

yep, no problem! i'll do my best to try to resemble the original. i'm happy to help! it just takes some time, so please bear with me .o. this is the recent piece, if you want to correct anything, feel free to add on! 

i also found out that her eyes need more shadowing and red lines. i added those, i'll work on the top of her head and pony tail now. the bangs are nearly finished, but i'll continue to work on it too.

**Comment by kkrosie123** [comment]
It looks amazing so far 💕 Please take your time, so excited to see what you finish with!

**Comment by PeachyPieeee** [comment]
Okay. I was going to reshape some of her body(such as the hands) as well, but if I do, I’ll open another issue again.

**Comment by PeachyPieeee** [comment]
mmkay, i have finished this one. i added some shading and recoloured the lineart on her body. here's the final.

![image](https://user-images.githubusercontent.com/66063309/173730855-b6d1bb00-938c-428c-a076-8daf29f40cdd.jpeg)

may i ask how do i send the psd?

**Comment by ThePotatoGuy** [comment]
Standard rules for hair changes are that the changes need to be applied to all officially released hairstyles, including our released spritepacks. That being said, the team likes your final version a lot so I think we can pull some of the work to making that happen. @kkrosie123 can you work with @PeachyPieeee to get this design applied to all of our hair styles? 

I tested the standard res versions and they look fine, I'll upload them to the discord later.

**Comment by kkrosie123** [comment]
Yeah, sure! There are a few bangs that may need some tweaks art-wise for application. Ie, a newer but not-yet-released low bun found in #9202 ~ there’s also the Shuchi’in Academy hairstyle, aka straight bangs, but I’m sure we could make it work.

**Comment by PeachyPieeee** [comment]
yep! i'll do my best ;) thanks for giving me a chance to contribute! i also adore you guys's work :3 if you guys want to directly contact me, dm me at discord: peachypieeee#3566 
just call me peachy!
